examples/prometheus/rules/kube.rules

```diff
@@ -0,0 +1,68 @@
+groups:
+- name: kubernetes-rules
+  rules:
+
+    - alert: DockerLatencyHigh
+      expr: max(kubelet_docker_operations_latency_microseconds{quantile="0.9"}) / 1e+06 > 1
+      for: 5m
+      labels:
+        severity: warning
+      annotations:
+        summary: Docker latency is high
+        description: "Docker latency is {{ $value }} seconds for 90% of kubelet operations"
+        alertType: latency
+        component: container runtime
+        selfHealing: false
+        url:
+
+    - alert: KubernetesAPIDown
+      expr: up{job="kubernetes-apiservers"} == 0
+      for: 10m
+      labels:
+        severity: critical
+      annotations:
+        summary: Kubernetes API server unreachable
+        description: "Kubernetes API server unreachable on {{ $labels.cluster }} instance {{ $labels.instance }}"
+        alertType: availability
+        component: kubernetes
+        selfHealing: false
+        url:
+
+    - alert: KubernetesAPIAbsent
```

Comment:
I guess this alert will also be firing in case prometheus isn't scraping metrics

Reply:
Right, this is to ensure we catch silent failures.